ACMILabs/playlist-label
# #78 Add the ability to align the collect arrow to the right side

Resolves #78 

Added the ability to optionally align the 'collect' element to the right.

### Acceptance Criteria
Replace acceptance criteria with the acceptance criteria in the ticket or check the box if none.
- [x] Ability to align the collect arrow to the right of screen

### Relevant design files
Add a links to the relevant design files or leave as none.
* https://trello.com/c/yPXFupsE/82-fix-location-of-collect-on-mpl-playlist-label

### Testing instructions
Add a list of steps required to test the feature.
1. Set your config.env to point to the MPL playlist
```
...
XOS_API_ENDPOINT=https://xos.acmi.net.au/api/
...
XOS_PLAYLIST_ID=119
...
LABEL_TEMPLATE=playlist.html
...
```
2. Set COLLECT_POSITION in config.env to 'right'
```
COLLECT_POSITION=right
```
3. Start your playlist with `cd development & docker-compose up`
4. Once the page loads, set the screen size to 1920x720
    - [Set screen size in chrome](https://ffeathers.wordpress.com/2016/04/16/set-size-of-browser-inner-window-for-screenshots-in-chrome/)
    - [Set screen size in safari](https://thesweetsetup.com/use-responsive-design-mode-safari/)
5. Note the 'collect' element is now in the bottom right of the screen
6. Emulate a tap with curl, notice the collect button registers a new 'tap' and looks correct
```
curl --location --request POST 'http://0.0.0.0:8081/api/taps/' --header 'Authorization: Token {{YOUR_XOS_TOKEN}}' --header 'Content-Type: application/json' --data-raw '{
    "lens": {
        "atr": "3b8f8001804f0ca0000003060300030000000068",
        "short_code": "xgdgqy"
    },
    "label": 659,
    "tap_datetime": "2018-11-13T16:02:26.565955+11:00",
    "data": {
        "lens_reader": {
            "mac_address": "00:00:00:00:00:09",
            "reader_ip": "127.0.0.1",
            "reader_model": "IDTech Kiosk IV",
            "reader_name": "nfc-1"
        }
    }
}'
```
7. Stop the playlist label with `ctrl + c`
8. Remove COLLECT_POSITION and start the app again as with step 3.
```
COLLECT_POSITION=
```
9. Notice the collect is back in its original position
10. Emulate a tap again, notice it looks correct

### DoD
For requester to complete:
~- [ ] New logic has appropriate unit tests~
~- [ ] Changelog has been updated if necessary~
~- [ ] Deployment / migration instruction have been updated if required~
